client/src/CoqDocument.ts

```diff
@@ -298,11 +299,11 @@ export class CoqDocument implements vscode.Disposable {
       }
       let stat : fs.Stats|undefined = undefined;
       try {
-        stat = fs.statSync(path.join(v, 'coqtop'));
+        stat = fs.statSync(path.join(v, coqtopExe));
       } catch(err) {}
       if(!stat && os.platform()==='win32') {
         try {
-          stat = fs.statSync(path.join(v, 'coqtop.exe'));
+          stat = fs.statSync(path.join(v, coqtopExe));
```

Comment:
Aren't we trying a second time the same thing here?

Reply:
Fixed; sorry for the delay.